.github/workflows/sync-bot.yml

```
@@ -13,34 +13,37 @@ on:
 
 jobs:
   sync_bot:
-    name: Sync Bot
+    name: Sync Bot Status Check
     if: ${{ !github.event.issue.pull_request }}
     runs-on: ubuntu-latest
     steps:
       - name: Set Up Deno
+        if: startsWith(github.ref_name, 'feature/')
         uses: denoland/setup-deno@v1
         with:
           deno-version: v1.x
 
       - name: Run Sync Bot (Issue Change)
+        if: startsWith(github.ref_name, 'feature/')
         run: |
           $scriptUrl = "${{ vars.SCRIPT_BASE_URL }}/${{ vars.CICD_SCRIPTS_VERSION }}/${{ vars.SCRIPT_RELATIVE_DIR_PATH}}/sync-bot-status-check.ts";
 
           $issueNumber = "${{ github.event.issue.number }}";
 
-          Write-Host "::notice::Project Name: ${{ vars.PROJECT_NAME }}";
-          Write-Host "::notice::Issue: $issueNumber";
+          Write-Host "::notice:Project Name: ${{ vars.PROJECT_NAME }}";
+          Write-Host "::notice:Issue: $issueNumber";
 
           if ($manuallyExecuted -and $issueNumber -eq "0") {
-            Write-Host "::error::The issue or PR number must be a value greater than 0.";
+            Write-Host "::notice:The issue or PR number must be a value greater than 0.";
```

Change this message type from a `notice` to and `error`

```suggestion
            Write-Host "::error::The issue or PR number must be a value greater than 0.";
```